<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakhberAbout</class>
 <widget class="QDialog" name="MakhberAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../main/appicons.qrc">
    <normaloff>:/appicon</normaloff>:/appicon</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="version_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Makhber XX.XX.XX-betaXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="release_date_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Released: MMM, DD YYYY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../main/appicons.qrc">
     <normaloff>:/applogo</normaloff>:/applogo</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>130</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="used_libraries">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>380</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Makhber was built with the fellowing libraries</string>
   </property>
  </widget>
  <zorder>release_date_label</zorder>
  <zorder>close_button</zorder>
  <zorder>version_label</zorder>
  <zorder>used_libraries</zorder>
 </widget>
 <resources>
  <include location="../main/appicons.qrc"/>
  <include location="../main/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>MakhberAbout</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
